skimage/filters/edges.py

```diff
@@ -170,6 +170,55 @@ def sobel_v(image, mask=None):
     return _mask_filter_result(result, mask)
 
 
+def get_sobel_weights(ndim):
+    """Produce Sobel kernel for axis 0.
+
+    Parameters
+    ----------
+    ndim : number of dimensions
+
+    Returns
+    -------
+    output : max(ndim,1)-dimensional array containing Sobel kernel in
+        direction of axis 0.
+
+    Notes
+    -----
+    If ndim < 0, then ndim is assumed to be 1.
+    To change the axis, use transpose(0, axis).
+    """
+    ret = np.array([1, 0, -1])
+    h = np.array([1, 2, 1])
+    for i in range(ndim - 1):
+        ret = np.multiply.outer(ret, h)
+    factor = ret.sum(1).max()  # Find maximum factor for normalization
```

Comment:
I'm not sure this is the right normalization but on the other hand the normalization of the 2D kernel seems weird (it does not correspond to the definition of the Wikipedia web page for Sobel)

Reply:
This was just a guess, based on what other kernels in `edges.py` look like. It's more of a placeholder.

Reply:
From git blame I think it was @ahojnnes who wrote the kernels with the normalization.